contrib/deploy_timesketch.sh

```diff
@@ -92,11 +92,8 @@ curl -s $GITHUB_BASE_URL/data/generic.mappings > timesketch/etc/timesketch/gener
 curl -s $GITHUB_BASE_URL/data/regex_features.yaml > timesketch/etc/timesketch/regex_features.yaml
 curl -s $GITHUB_BASE_URL/data/winevt_features.yaml > timesketch/etc/timesketch/winevt_features.yaml
 curl -s $GITHUB_BASE_URL/data/ontology.yaml > timesketch/etc/timesketch/ontology.yaml
-curl -s $GITHUB_BASE_URL/data/sigma_rule_status.csv > timesketch/etc/timesketch/sigma_rule_status.csv
-curl -s $GITHUB_BASE_URL/data/tags.yaml > timesketch/etc/timesketch/tags.yaml
```

Comment:
Why also remove the `tags.yaml` file?  IIRC this still exists and should be used by the Tagger Analyzer. (https://github.com/google/timesketch/blob/master/data/tags.yaml)

Reply:
My fault, it says in the initial comment: Removing a duplicate.